### PR TITLE
[Merged by Bors] - Added helpful adders for systemsets

### PR DESCRIPTION
**Pull request by CGMossa**
# Objective

- This adds a way to add `SystemSet`s to Apps.

### CODE REVIEW

File: crates/bevy_app/src/app_builder.rs [hunk 1]
```diff
@@ -232,6 +232,11 @@ impl AppBuilder {
         self.add_startup_system_to_stage(StartupStage::Startup, system)
     }
 
+    pub fn add_startup_system_set(&mut self, system_set: SystemSet) -> &mut Self {
+        self.add_startup_system_set_to_stage(StartupStage::Startup, system_set);
+        self
```
Comment (bjorn3) [inline crates/bevy_app/src/app_builder.rs:237]:
```suggestion
        self.add_startup_system_set_to_stage(StartupStage::Startup, system_set)
```

**Review by alice-i-cecile — APPROVED** [review]

**Comment by alice-i-cecile** [comment]
I still think we should move to a better builder API for this sort of thing, but this is necessary until then.

**Review by NathanSWard — APPROVED** [review]
> I still think we should move to a better builder API for this sort of thing, but this is necessary until then.

Yep agreed.

**Review by mockersf — APPROVED** [review]

**Review by lafi-99 — APPROVED** [review]

**Review by lafi-99 — APPROVED** [review]

**Pull request closed without merging**